### ISSUE TITLE
package maintenance 

### ISSUE THREAD

**Issue by isaacsas** [kaandocal/FiniteStateProjection.jl#13]
@kaandocal would you consider putting this under SciML so we could get more people access to maintain it? You could still have full repo permissons, but having it in an org might help with keeping dependencies updated. Just a thought!

**Comment by kaandocal** [comment]
Sure, happy to do that - I've not been very up to date with some of the more recent changes to Catalyst and Symbolics, as you might have noticed...

**Comment by isaacsas** [comment]
No worries! I'm trying hard to keep Catalyst v13 stable and avoid any breaking changes for a while. It was a nice improvement I think and will ultimately allow full (spatial too!) model specification within DSL, but it was breaking. I can't control Symbolics, but the last breaking release had noticeable performance improvements, so I'd say it was worthwhile.

I'll ping you here once you have been added to SciML and can initiate the repo transfer.

**Comment by isaacsas** [comment]
I just sent you an invite to join SciML. Once you accept you can transfer this repo to the org. I'll make sure you still have full write permissions on it.

**Comment by kaandocal** [comment]
I appreciate the work on both packages. Unfortunately the invitation link has expired as I had other commitments for the past week or two - if you could please re-send it I will make sure to transfer the repo. Thanks and sorry for the bother!

**Comment by isaacsas** [comment]
@kaandocal I just sent you another invitation.

**Comment by isaacsas** [comment]
Sorry for the delay, I was traveling for a week and am still catching up myself on stuff...

**Comment by kaandocal** [comment]
No worries, the hold-up was on me. Done!

**Comment by isaacsas** [comment]
Yup, I see it is now working. Is there any reason not to make a new v.3 release with the updated Catalyst support? (i.e. are you aware that there are open issues?)

**Comment by isaacsas** [comment]
@kaandocal is it ok to make a release of the current version to get the Catalyst 13 support out there?

**Comment by sallyseal** [comment]
@kaandocal @isaacsas is FiniteStateProjection.jl compatible with Catalyst 13 now? I'm having an issue with using symbolic stoichiometry matrices. Thanks!

**Comment by isaacsas** [comment]
It looks like it never got a release for Catalyst 13, but I just started the process.

**Comment by sallyseal** [comment]
> It looks like it never got a release for Catalyst 13, but I just started the process.

Thanks @isaacsas - will check back

**Comment by kaandocal** [comment]
@sallyseal The delay is my doing - I mistakenly assumed the updated version would be reflected in the registry automatically.

**Comment by kaandocal** [comment]
@sallyseal Please let me know if this works for you. Seems like not updating in time created a big mess...
